compose/runtime/runtime/src/nonEmulatorCommonTest/kotlin/androidx/compose/runtime/CompositionTests.kt

```diff
@@ -4353,20 +4353,22 @@ class CompositionTests {
         revalidate()
     }
 
-    @Test
-    fun composableDelegates() = compositionTest {
-        val local = compositionLocalOf { "Default" }
-        val delegatedLocal by local
-        compose {
-            Text(delegatedLocal)
-
-            CompositionLocalProvider(local provides "Scoped") { Text(delegatedLocal) }
-        }
-        validate {
-            Text("Default")
-            Text("Scoped")
-        }
-    }
+// commented because of compilation failure on k2.1.21 (https://youtrack.jetbrains.com/projects/KT/issues/KT-77508)
+// should be restored with migration to k2.2
+//    @Test
```

Comment:
Add `@Ignore` instead of commenting out. Also, it's better to ignore it only for specific platforms

Also, common runtime tests are supposed to be in AOSP now. I'll add this ignoring in https://r.android.com/3642444 too

Reply:
seems like presence of that function(even `@Ignore`d) fails other tests in that file

Reply:
please see my comment here - https://youtrack.jetbrains.com/issue/CMP-8252/Update-Kotlin-to-2.1.21-in-Compose-Multiplatform-core-repo#focus=Comments-27-12068766.0-0

we need to comment it for now.

There was a change in kotlin 2.1.20 for native targets only. Then the same change was applied for k/js targets, but not for k/wasm yet. It's fixed in 2.2.0-RC2.